### PR TITLE
Apply cookiecutter-wagtail-package

### PR DESCRIPTION
**Pull request by allcaps**

### CODE REVIEW

**Comment by zerolab** [comment]
@allcaps any chance you could split into at least 2 commits.
1. adding the cookiecutter changes
2. the linting

as it stands, got 1623 files to look at :D

**Comment by allcaps** [comment]
What I did:

- Run [the cookiecutter](https://github.com/wagtail/cookiecutter-wagtail-package).
- Copied all templates from the old project into the fresh new project. They are in `templates/wagtailfontawesomesvg` 
- Copied over the `.git` directory (to keep the git history)
- Commit the changes. Yes, one big ugly commit.

Note, this is a very simple package. It only ships templates, nothing else. So all changes are the coockiecutter defaults. I assume they are correct. If not, we'd discover that soon enough.

The things I'd like to figure out are:

1. Adding tests and run the CI/CD
2. Deployment with PyPi keys

The coockiecutter supplies `.circleci` and `.github/workflows`. Not sure what I need to do to make them work. I'll look into that.

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  pull_request:
+
+jobs:
+  test-sqlite:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python: ['3.7', '3.8', '3.9']
+        django: ['3.0', '3.1', '3.2']
+        wagtail: ['2.14', '2.15']
```
Comment (zerolab) [inline .github/workflows/test.yml:18]: note to self: we'll need to update this to be 2.15, 4.0 and 4.1. And Django targets accordingly

Comment (zerolab) [reply .github/workflows/test.yml:18]: will submit a separate PR once https://github.com/wagtail/cookiecutter-wagtail-package/pull/28, https://github.com/wagtail/cookiecutter-wagtail-package/pull/29 and https://github.com/wagtail/cookiecutter-wagtail-package/pull/32 are in

File: README.md [hunk 2]
```diff
@@ -5,8 +5,25 @@ Add [Font Awesome 5.14](https://fontawesome.com/icons?d=gallery&m=free) SVG icon
...
+- [Changelog](https://github.com/allcaps/wagtail-font-awesome-svg/blob/main/CHANGELOG.md)
+- [Contributing](https://github.com/allcaps/wagtail-font-awesome-svg/blob/main/CHANGELOG.md)
+- [Discussions](https://github.com/allcaps/wagtail-font-awesome-svg/discussions)
+- [Security](https://github.com/allcaps/wagtail-font-awesome-svg/security)
+
+## Supported versions
+
+- Python ...
+- Django ...
+- Wagtail ...
```
Comment (zerolab) [inline README.md:24]: note to self: one to revisit once the other tweaks are in place

File: setup.py [hunk 3]
```diff
@@ -1,41 +1,48 @@
...
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Programming Language :: Python :: 3.9",
+        "Framework :: Django",
+        "Framework :: Django :: 3.0",
+        "Framework :: Django :: 3.1",
+        "Framework :: Django :: 3.2",
+        "Framework :: Wagtail",
+        "Framework :: Wagtail :: 2",
```
Comment (zerolab) [inline setup.py:41]: We most certainly want at least `'Framework :: Wagtail :: 4',`

**Review by zerolab — COMMENTED** [review]
@allcaps sorry this took so long.

The changes themselves look fine to me.
I do have a few issues/nitpicks with the fact cookiecutter is not behind which means the changes are a bit of a regression from what you currently have.

Up to you if you want to merge this, but imho we want to update cookiecutter first and do it nice and clean here, even if it means redoing the work.

Will get back to you on the tests. https://github.com/wagtail/cookiecutter-wagtail-package/pull/29 is how we can do automated PyPI publishing.

**Pull request closed without merging**